eq-author/src/components/ContentPickerv2/DestinationPicker/DestinationEnd.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { colors } from "constants/theme";
+
+import {
+  SectionTitle,
+  SubMenuItem,
+  MenuItemTitles,
+  MenuItemTitle,
+  MenuItemSubtitle,
+} from "../Menu";
+
+import IconPage from "./icon-page.svg?inline";
+
+const EndTitle = styled(SectionTitle)`
+  border-top: 1px solid ${colors.lightGrey};
+  border-bottom: 1px solid ${colors.lightGrey};
+`;
+
+const StyledMenuItemTitles = styled(MenuItemTitles)`
+  display: flex;
+  align-items: center;
+`;
+
+// eslint-disable-next-line no-unused-vars
+export const Icon = styled(({ isActive, ...rest }) => <IconPage {...rest} />)`
+  margin-right: 1em;
+  .path {
+    fill: ${props => (props.isActive ? "white" : colors.darkGrey)};
+  }
+`;
```

Comment:
Do we need to pass isActive in here (line 27) in order for line 30 to work?

Reply:
Good question. This is a fix to prevent `isActive` being passed down to the svg dom element. `isActive` is not a valid svg attribute, so it was throwing errors. What line 27 does is destructure `isActive` from the props, and we only pass the `rest` of them to IconPage. Thus removing the unwanted prop but still being able to use it for styling.

Reply:
ahhh - clever!